examples/advanced-anchors.html

```diff
@@ -78,7 +78,9 @@
               ]);
               rl.noFill().stroke = ll.noFill().stroke = editColor;
 
-              letter.add(rl, ll, p, l, r);
+              var g = two.makeGroup(rl, ll, p, l, r);
+              g.translation.addSelf(polygon.translation);
+              letter.add(g);
```

Comment:
Anchors in new Two.js seem to have a different coordinate system. This may be worth revisiting--I'm not sure how janky my solution is.

Reply:
That's correct. Between the current build and the last version running on the website `Two.Anchor`s added `Two.Anchor.relative` which defines whether the anchor should be considered with relative space to its parent shape or in world space. Thanks for adding the `addSelf`